File: core/src/frontendServer/server/models/PistolPickup.js
const Pickup = require('./Pickup');
const constants = require('../settings/constants');
const ProjectileType = require('./ProjectileType');

class PistolPickup extends Pickup {
    constructor(x, y) {
        super(x, y, constants.PISTOL_SPRITE_WIDTH, constants.PISTOL_SPRITE_HEIGHT, ProjectileType.PISTOL)
    }
}

module.exports = PistolPickup;

File: core/src/com/mygdx/game/model/PistolPickup.kt
package com.mygdx.game.model

import com.badlogic.gdx.graphics.Texture
import com.mygdx.game.settings.PISTOL_SPRITE_HEIGHT
import com.mygdx.game.settings.PISTOL_SPRITE_WIDTH

class PistolPickup(x: Float = 0f, y: Float = 0f, texture: Texture) : Pickup(x, y, PISTOL_SPRITE_WIDTH, PISTOL_SPRITE_HEIGHT, texture)